shared/templates/sysctl/tests/wrong_usr_lib_correct_etc.pass.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Thank you for the tests.
I suggest you  also add a test for `wrong_usr_lib_wrong_etc.fail.sh`